Odata-docs/odatalib/v7/edm/iedmtargetpath-for-annotations-target.md

```diff
@@ -0,0 +1,463 @@
+---
+title: "Use IEdmTargetPath for out-of-line annotations target"
+description: "Use IEdmTargetPath for out-of-line annotations target"
+author: kenitoinc
+ms.author: kemunga
+ms.date: 05/2/2023
+ms.topic: article
+ 
+---
+# Use IEdmTargetPath for out-of-line annotations target
+**Applies To**: [!INCLUDE[appliesto-odataclient](../../../includes/appliesto-odatalib-v7.md)]
+
+The Edm library lets you [define annotations](/odata/odatalib/edm/define-annotations).
+
+[This](/odata/odatalib/edm/set-annotations-schema) documentation describes how to define **out-of-line** annotations that targets a model element.
+
+## Background
+
+Below is a simple Edm model with an out-of-line annotation that targets a model element.
```

Comment:
Consider the follow metadata describing a service where a model element `Employee` defined in the `Ns` namespace has annotations defined out-of-line in the `Default` namespace: